config/jobs/kubernetes/sig-aws/eks/eks-periodics.yaml

```diff
@@ -73,53 +73,3 @@ periodics:
       - --eks-nodes=100
       - --test_args=--ginkgo.focus=\[Feature:Performance\] --minStartupPods=8
       - --timeout=300m
-
-# Kubernetes e2e correctness tests against EKS 1.10 build.
```

Comment:
The removal of this might raise eyebrows, but I get it, this isn't something we as a community support.

Reply:
I think eks can still run 1.10 CI (if they care) in their prow instance, instead of community prow instance :-) cc @gyuho @shyamjvs